scripts/_common.sh

```diff
@@ -377,3 +311,122 @@ ynh_remove_extra_repo () {
 	# Update the list of package to exclude the old repo
 	ynh_package_update
 }
+
+# Install packages from an extra repository properly.
+#
+# usage: ynh_install_extra_app_dependencies --repo="repo" --package="dep1 dep2" [--key=key_url] [--name=name]
+# | arg: -r, --repo - Complete url of the extra repository.
+# | arg: -p, --package - The packages to install from this extra repository
+# | arg: -k, --key - url to get the public key.
+# | arg: -n, --name - Name for the files for this repo, $app as default value.
+ynh_install_extra_app_dependencies () {
```

Comment:
where do we use this helper?

Reply:
it's used nowhere but it's part of the https://github.com/YunoHost-Apps/Experimental_helpers/blob/master/ynh_add_secure_repos/ynh_add_secure_repos__3, so I copied/pasted all the content of the helper